CONTRIBUTING.md

```diff
@@ -127,6 +127,25 @@ The conventional commit type (in PR title) is very important to automatically bu
 
 There is no need to update the CHANGELOG in a PR because it will be updated as part of the release process (see [RELEASING.md](RELEASING.md) for more details).
 
+### Testing
+
+Most unit tests case be run via:
+
+```sh
+npm test
+```
+
+However, some instrumentations require some test-services to be running (e.g. a MongoDB server to test the `instrumentation-mongodb` package). Use the `test-services`-related npm scripts to start test services in Docker and to run the tests with the appropriate configuration to use them:
+
+```sh
+npm run test-services:start     # starts services in Docker
+npm run test:with-test-services # runs 'npm test' with envvars from test/test-services.env
+npm run test-services:stop      # stops Docker containers
+```
+
+This set of commands works in the top-level directory to test all packages, or
+in a specific package directory that requires a test service (e.g. `plugins/node/opentelemetry-instrumentation-mongodb`).
```

Comment:
```suggestion
in a specific package directory to test only that specific package (e.g. `plugins/node/opentelemetry-instrumentation-mongodb`), regardless if the tests need a test service.
```

Reply:
> regardless if the tests need a test service.

Actually that wasn't my intent. Packages that have no dep on a test-service won't typically have the `npm run test:with-test-services` script.

Reply:
ah I see, the wording made me think that was the case, meaning I thought the `set of commands` was referring to the `start -> with-test-services -> stop`, but I guess by set you meant the group of 3 AND the `npm start`. Can you make this more clear? So something along the lines of:

```
The first option can be used in the top-level directory to test all packages, 
while the second option can be executed from specific package directory that 
requires a test service.
```

And also add some clarification as: 
```
if you run the first option from the top-level, the tests with required services will
(or will not? I'm not clear about that, so the clarification will definitely help here hehe) 
start their required services and be executed.
```

Reply:
Er, now I'm confused. :) I was attempting to say that running all three of 

```
npm run test-services:start     # starts services in Docker
npm run test:with-test-services # runs 'npm test' with envvars from test/test-services.env
npm run test-services:stop      # stops Docker containers
```

works if one is in the top-directory of the git clone. And they also work if cd'd into a specific package of the monorepo if, say, you are doing development just on that package.

Reply:
Ah I see, so it needs some rewording, I was confused 😅